recipes/hyperscan/all/conanfile.py

```diff
@@ -41,27 +45,40 @@ class HyperscanConan(ConanFile):
     }
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _minimum_compilers_version(self):
+        return {
+            "apple-clang": "5.1",
+            "clang": "3.4",
+            "gcc": "4.9",
+            "intel": "15",
+            "sun-cc": "5.14",
+            "Visual Studio": "12"
+        }
 
     @property
     def _build_subfolder(self):
         return "build_subfolder"
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        os.rename("hyperscan-{0}".format(self.version), self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def build_requirements(self):
-        self.build_requires("ragel/6.10");
+        self.build_requires("ragel/6.10")
 
     def requirements(self):
-        self.requires("boost/1.79.0");
+        self.requires("boost/1.79.0")
         if self.options.build_chimera:
             self.requires("pcre/8.45")
 
     def validate(self):
-        tools.check_min_cppstd(self, "11")
+        if self.settings.compiler.get_safe("cppstd"):
```

Comment:
```suggestion
        if self.info.settings.compiler.get_safe("cppstd"):
```
It's part of the migration guide: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-validate-method